website/src/pages/ErrorPage/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, Grid, Typography } from "@material-ui/core";
+import React from "react";
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import Footer from "../../components/Footer";
+import { useHistory } from "react-router-dom";
+
+
+const ErrorPage: React.FC = () => {
+    const { t } = useTranslation();
+    const classes = useStyles();
+    const history = useHistory();
+    return (
+        <div className={classes.root}>
+            <div className={classes.wrapper}>
+                <Grid
+                    container
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
+                >
+                    <Grid item xs={12} lg={12}>
```

Comment:
```suggestion
                    <Grid item xs={12}>
```
lg={12} is not required. xs={12} will take 12 for the devices above it i.e., md and lg